packages/core/src/sass/app/_common/components/account-switcher.scss

```diff
@@ -194,6 +194,11 @@
             height: 100%;
             border-radius: 0;
         }
+        & .dc-themed-scrollbars {
+            &::-webkit-scrollbar-thumb {
+                background-color: var(--text-less-prominent);
```

Comment:
Is there any reason why we change scrollbar background color here?

Reply:
yes because the guys who reported it, wanted it in darker color because it was not that much visible.